crates/protect-ffi/Cargo.toml

```diff
@@ -11,11 +11,15 @@ crate-type = ["cdylib"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-cipherstash-client = "0.18.0"
+cipherstash-client = "0.22.0"
 hex = "0.4.3"
 neon = "1"
 once_cell = "1.20.2"
 serde = "1.0.218"
 serde_json = "1.0.139"
 thiserror = "2.0.8"
 tokio = { version = "1", features = ["full"] }
+# cipherstash-client specifies winnow 0.6.20 in its deps, but uses exports from `winnow::prelude`
+# that aren't available until later versions. 0.6.26 is what cipherstash-client uses in its lockfile
+# at the time of this change. Using 0.6.26 prevents compiler errors due to missing exports in 0.6.20.
+winnow = "0.6.26"
```

Comment:
I'll take a look at fixing this up in `cipherstash-client` and will remove the `winnow` dep here after (since I don't think it's worth blocking this change on that fix).